### ISSUE TITLE
Output per request stats in addition to global_stats.json

### ISSUE THREAD

**Issue by ingojaeckel** [gatling/gatling#2482]
At the end of a simulation, gatling persists the global statistics (overall 95th percentile, mean response time, etc) into js/global_stats.json. The per request metrics are written into js/stats.js instead.

js/global_stats.json:
```javascript
{
    "name": "Global Information",
    "numberOfRequests": {
        "total": 6,
        "ok": 6,
        "ko": 0
    },
    ...
}
```

js/stats.js:
```javascript
var stats = {
  "type": "GROUP",
  "name": "Global Information",
  "path": "",
  "pathFormatted": "missing-name-b06d1",
  "stats": { /* the same stats that are in global_stats.json */ },
  "contents": {
    "some-request-b5369": {
      "type": "REQUEST",
      "name": "Some Request,
      "path": "Some Request",
      "pathFormatted": "some-request-b5369",
      "stats": {
        "name": "Some Request",
        "maxResponseTime": {
          "total": "378",
          "ok": "378",
          "ko": "-"
        },
        // also 95th percentile, mean, min response time, etc
      }
    }
  }
}
```

To simplify post processing the per-request statistics, I would like to replicate the JSON data from stats.js into a JSON file. If Gatling would produce a stats.json file (similar to global_stats.json), it would be easier accessible to tools like Gatling's Jenkins plugin. Right now this plugin only opens the global_stats.json file and creates a trend of overall response times.

**Comment by pdalpra** [comment]
closed by 4a4b970be3de8d21471c1ad0f425fc9cf0be3095

**Comment by asbaghel** [comment]
How to save stats.js (Report) in influxDB
